### ISSUE TITLE
Does this works

### ISSUE THREAD

**Issue by uhlhosting** [MatthewKuKanich/ESP32-AirTag-Scanner#1]
Can it work with standard flipper zero dev boards?

**Comment by MatthewKuKanich** [comment]
> Can it work with standard flipper zero dev boards?

No, the official dev board doesn't have BLE functionality. You'll need an ESP32-WROOM, ESP32-S3, or other ESP32 board that has integrated BLE.
